packages/react/src/primitives/Tabs/types.ts

```diff
@@ -38,6 +38,7 @@ export interface BaseTabsProps extends BaseTabsListProps {
   items?: Array<{
     value: string;
     isDisabled?: boolean;
+    groupID: string;
```

Comment:
Think this is a breaking change?

Reply:
I could be wrong, but I don't think so. I have `groupID` initialized as an empty string in TabsContext, and there is no user interaction with `groupID` when creating tab groups.

Reply:
Think requiring a new prop in the `items` definition will break existing usage? Example:
```
// current required properties
<Tabs items={[{ content: 'content', label: 'label', value: 'value' }]} />

// updated required properties
<Tabs items={[{ content: 'content', groupId: 'groupId', label: 'label', value: 'value' }]} />
```

Also `groupId` is an internal detail and not something that we want to expose as overridable right?